applications/EigenSolversApplication/README.md

```diff
@@ -126,23 +126,22 @@ In case you have installed [MKL](https://software.intel.com/en-us/mkl), you can
 
 ## Install MKL on Ubuntu with apt
 
-Open a terminal window and run the following commands to install MKL from the official Intel repository:
+Open a terminal window and run the following commands to install MKL from the official Intel repository.
+For more details see https://software.intel.com/en-us/articles/installing-intel-free-libs-and-python-apt-repo.
 
 ```bash
-apt-get update -y
-apt-get upgrade -y
-apt-get install -y gnupg2 software-properties-common wget
-wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB -P/tmp
-apt-key add /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-rm /tmp/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB
-echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list
```

Comment:
@oberbichler this command did not work for me

Reply:
i replaced it with the one from the intel page

Reply:
The other one is also from the intel page